CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -196,35 +196,49 @@ extension GiniScreenAPICoordinator {
     @objc func back() {
         switch screenAPINavigationController.topViewController {
         case is CameraViewController:
-            trackingDelegate?.onCameraScreenEvent(event: Event(type: .exit))
-            // TODO: rethink this when you add Camera access screen events!
-            // is also accessed from that screen
-            // do we need a different name for the screen in that case?
-            AnalyticsManager.track(event: .closeTapped, screenName: .camera)
-
-            if pages.type == .qrcode {
-                screenAPINavigationController.dismiss(animated: true)
-            }
-
-            if pages.count > 0 {
-                if screenAPINavigationController.viewControllers.count > 1 {
-                    screenAPINavigationController.popViewController(animated: true)
-                } else {
-                    screenAPINavigationController.dismiss(animated: true)
-                }
-            } else {
-                closeScreenApi()
-            }
+            handleCameraViewControllerBack()
         case is AnalysisViewController:
-            trackingDelegate?.onAnalysisScreenEvent(event: Event(type: .cancel))
-            AnalyticsManager.track(event: .closeTapped, screenName: .analysis)
-            screenAPINavigationController.dismiss(animated: true)
+            handleAnalysisViewControllerBack()
```

Comment:
@ValentinaIancu-Gini 
could you please re-think the naming, I have a feeling that something is missing after `back`?
maybe something like `navigateBackFromAnalysisViewController`
or `handleBackNavigationFromAnalysis`?

Please consider renaming for the `handleDefaultBack()` as well

Reply:
I will rename.